src/averageAges.js

```diff
@@ -1,64 +1,63 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+const FEMALE = 'f';
+const MALE = 'm';
+
+function sumAges(cadaver) {
```

Comment:
`getAvarageAge` is a better name for the function
Also, try to make up a better name, not `cadaver`